spec/factories/vacations.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :vacation do
+    start_date { Date.today }
+    end_date { 2.months.from_now }
+    status { 1 }
```

Comment:
```suggestion
    status { Vacation.statuses[:pending] }
```
I'm not sure if this will work but it would be best if you could just reference the status as `pending` without depending on the number.

I got the code from here: https://www.rubyinrails.com/2019/04/12/how-to-get-integer-value-from-enum-in-rails/